dev/ctests/cases/C48_S2SW_gfs_fcst_seg0.yaml

```
@@ -7,44 +7,541 @@
-# Note: The below three files is only an example. The cmpfiles tag means it will use checksum to compare the files in the two locations.
-# Other extensions can be added and/or folded into the FileUtils class in wxflow FileUtilities.py
-#
-#output_files:
-#    cmpfiles:
-#        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/{{ 'TEST_NAME' | getenv }}/output_files/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table"]
-#        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/{{ 'TEST_NAME' | getenv }}/output_files/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml"]
-#        - ["{{ 'CTESTS_STAGED_TESTS_DIR' | getenv }}/{{ 'TEST_NAME' | getenv }}/output_files/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure", "{{ 'RUNTESTS' | getenv }}/COMROOT/{{ 'TEST_NAME' | getenv }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure"]
+output_files:
+    cmpfiles:
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.MOM_input, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.MOM_input]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.diag_table]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ice_in, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ice_in]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.input.nml]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.model_configure]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ufs.configure, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ufs.configure]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ww3_shel.nml, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/conf/ufs.ww3_shel.nml]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf000.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf000.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf003.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf003.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf006.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf006.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf009.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf009.txt]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf0{{ cyc }}.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf0{{ cyc }}.txt]
```

Why is this atm.logf0{{ cyc }} and not this atm.logf012?
```suggestion
        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf012.txt, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/history/gfs.t{{ cyc }}z.atm.logf012.txt]
```